sdk/src/test/scala/com/horizen/integration/SidechainStateIntegrationTest.scala

```diff
@@ -349,4 +352,141 @@ class SidechainStateIntegrationTest
         sidechainState.closedBox(b).isDefined)
     }
   }
+
+  @Test
+  def applyModifierWithCSWDisabled(): Unit = {
```

Comment:
From my point of view, keeping all the checks is a way to verify and to document that everything is exactly the same, with or without CSW, except for the utxomerkletree. Maybe I can refactor the tests in order to put all the common parts in a common method and leave outside only the testing of the utxomerkletreeroot, this way it will be more explicit.
What do you think?

Reply:
Right, agree.